### PR TITLE
Improve performance.

### PR DESCRIPTION
**Pull request by Garbee**
This merge does a few things to improve the performance:

* Dump jQuery, it isn't doing anything that can't be done with raw JS pretty easily.
* Trim down the markup in a few places to remove unneeded DOM nodes.
    * Particularly the `#footer` div wrapping the footer element and `.wrapper` is now just `body`
* Fix CSS to work with the DOM structure.

With this I also just dumped the minified CSS file. It isn't providing much of a benefit over just gzipping the contents going across the wire. This helps keep things easier to maintain going forward since a minified file doesn't need to be handled and committed.

Since now we just have two resource files, I also dumped the `css` and `js` sub-folders since they don't provide anything towards clarity in the structure. 4 files in one folder shouldn't be hard to understand. This also saves a few measly bits in the URLs going across the wire.

Changes to the JS are:

* Creates the filter form on-the-fly. So if JS isn't available it is never shown.
* Filters only by the first column (file/folder names) instead of all columns. This doesn't let developers filter by size or date anymore. If those features are desired I can handle it in a way that not everything is filtered at once since that could lead to undesirable results.

With these changes, the total return on a page is roughly 1KB on small directory lists. Where just the jQuery inclusion before was ~92KB. So about a 90 fold reduction in bandwidth for the initial response.

### CODE REVIEW

**Comment by Naereen** [comment]
Wow, that's impressive! :1st_place_medal:
I will test it and merge if it looks good (I'm sure it does)

**Comment by Garbee** [comment]
The only caution note (which I just remembered) is IE/Edge support. They don't support the `ChildNode.after()` API in JS. So if we *need* support for those then I would need to tweak the form insertion JS a bit. I just used the new API being quick and clean in the refactor.

**Comment by Naereen** [comment]
I honestly don't care about support for IE.

**Comment by Garbee** [comment]
You may want to peek at [my current setup](https://static.untamedworlds.com). I just went though and basically rethemed from scratch based on the [nord color scheme](https://github.com/arcticicestudio/nord). If you want to go dark I can PR these changes in. It also has a little extra JavaScript for checking the current query parameter and assigning a `current` class where needed to rotate the arrow to help indicate what is being filtered by.

**Comment by Naereen** [comment]
> "If you want to go dark"

→ i :rofl: ?

**Comment by Naereen** [comment]
Thanks again, :+1: !

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ The fancyindex module can be found [here](https://github.com/aperezdc/ngx-fancyi
...
 
 1. Make sure you have the fancyindex module compiled with nginx, either by compiling it yourself or installing nginx via the full distribution (paquet ``nginx-extras``).
-2. Include the content of [fancyindex.conf](fancyindex.conf) in your location directive (``location / {.....}``) in your nginx config (usually ``nginx.conf``).
-3. Move the ``Nginx-Fancyindex-Theme/`` folder to the root of the site directory.
-4. Restart/reload nginx.
-5. Check that it's working, and enjoy!
+1. Include the content of [fancyindex.conf](fancyindex.conf) in your location directive (``location / {.....}``) in your nginx config (usually ``nginx.conf``).
+1. Move the ``Nginx-Fancyindex-Theme/`` folder to the root of the site directory.
+1. Restart/reload nginx.
+1. Check that it's working, and enjoy!
```
Comment (Garbee) [inline README.md]: With markdown you don't literally number items. You always just use `1.` to indicate that it is a part of an ordered list. The parser then does the number properly on its side. It saves you from changing numbers all the time if things change. Even if it does look odd.

Comment (Naereen) [inline README.md]: I know that the syntax authorize this, but for non-rapidly changing documents it's a pointless optimization. The source looks better with 1., 2., 3., than 1., 1., 1., ..;

**Pull request merged**